proto/collection.proto

```diff
@@ -1,138 +1,150 @@
-// blueprint.proto
+// collection.proto
 syntax = "proto3";
 
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/duration.proto";
 
+package com.gu.mobile.mapi.models;
```

Comment:
I guess because we specify the `package` we don't need the optional `java_package`

Reply:
Also because the "swift_prefix" option ignores the package for Swift.